snappy_pipeline/workflows/variant_calling/__init__.py

```diff
@@ -183,6 +183,9 @@
 # Default configuration variant_calling
 step_config:
   variant_calling:
+    baf_file_generation:
+      min_ad: 10
```

Comment:
What is this argument and where are you using it?

Reply:
Good point, adding note and actually using it.